README.md

```diff
@@ -1,4 +1,4 @@
-![PyPI](https://img.shields.io/pypi/v/pyuniqid) ![PyPI - Downloads](https://img.shields.io/pypi/dm/pyuniqid)
+[![PyPI](https://img.shields.io/pypi/v/pyuniqid)](https://pypi.org/project/pyuniqid) [![PyPI - Downloads](https://img.shields.io/pypi/dm/pyuniqid)](https://pypi.org/project/pyuniqid/#files) [![License](https://img.shields.io/github/license/boriskurikhin/pyuniqid)](https://raw.githubusercontent.com/boriskurikhin/pyuniqid/master/LICENSE)
```

Comment:
Added links that direct viewers to the right place when you click a badge, also included a license badge, as appose to embedding the license directly into the bottom of the README.